docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -241,18 +242,32 @@ Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post) endpoi
 We also provide the ability to add any FHIR resources to a Patient with our [Create Patient's Consolidated
 Data](/medical-api/api-reference/fhir/create-patient-consolidated) endpoint.
 
-## 9. Share Medical Data with the Networks 🚀🚀🚀
+## 9. Generate Patient Medical Record Summary ⭐⭐⭐
```

Comment:
I'd just condense this section with a TL;DR of why this is great and why someone would want to do this, and link to the guide at `docs/medical-api/more-info/medical-record-summary.mdx` instead of duplicating info

We want the docs to be as concise as possible, especially the quickstart guide